### PR TITLE
Create DRAFT patching post to mull-ovrr with Andre and CoE

### PR DESCRIPTION
**Pull request by wpschaub**

### CODE REVIEW

**Comment by wpschaub** [comment]
@MerlinDarkstar can you review and suggest improvements? Once published we can cross reference with our patch-management initiative.

**Review by MerlinDarkstar — APPROVED** [review]

File: content/what-keeps-us-awake-at-night-patching.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+# Toxic server debt
+
+The first is the iron, the servers, that host the outcome of our software development. 
+
+> ![Cloud Accountability](../images/what-keeps-us-awake-at-night-patching-1.png)
+> Image Source: [Shared responsibility in the cloud](https://learn.microsoft.com/en-us/azure/security/fundamentals/shared-responsibility) 
+
+The image, I loaned from a Microsoft article, clearly shows that we are accountable for our servers and that we take responsibility to keep them up-to-date and secure. As we move to platform as a service (PaaS) or software as a service (SaaS) we delegate the accountability to the hosting vendor.
```
Comment (MerlinDarkstar) [inline content/what-keeps-us-awake-at-night-patching.md]: Platform-as-a-Service (PaaS)
and 
Software-as-a-Service (SaaS)

I think there should be dashes in there.

**Review by MerlinDarkstar — DISMISSED** [review]

File: content/what-keeps-us-awake-at-night-patching.md (hunk 1 above)
Comment (wpschaub) [reply content/what-keeps-us-awake-at-night-patching.md]: Correct, fixing it now.

**Review by danielrbroderick — DISMISSED** [review]
Looks good

**Review by MerlinDarkstar — DISMISSED** [review]
Looking good

**Comment by wpschaub** [comment]
@AndreKaminski any suggested or required updates? How can we improve the story?

**Review by AndreKaminski — COMMENTED** [review]
could you please remove 'vulnerabilities' from this statement "I am currently having sleepless nights with technical debt and vulnerabilities in two areas.". I don't want anyone to get any ideas of trying to find those.
Also, there is one more way, which I like - continuous dedication of portion of capacity every sprint for tech debt stories.

**Review by danielrbroderick — DISMISSED** [review]

**Comment by wpschaub** [comment]
> could you please remove 'vulnerabilities' from this statement "I am currently having sleepless nights with technical debt and vulnerabilities in two areas.". I don't want anyone to get any ideas of trying to find those. Also, there is one more way, which I like - continuous dedication of portion of capacity every sprint for tech debt stories.

@AndreKaminski  I made the change. Please review and advise.

PS: I believe that Option 3 - Continuous Housekeeping is aligned with your way.

Will plan to get this post published on Wednesday and yours on Friday.

**Review by AndreKaminski — APPROVED** [review]
Looks good, thank you

**Review by danielrbroderick — APPROVED** [review]

**Review by MerlinDarkstar — APPROVED** [review]

**Pull request merged**